tests/Middleware/MiddlewareTestCase.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of Raven.
+ *
+ * (c) Sentry Team
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sentry\Tests\Middleware;
+
+use PHPUnit\Framework\TestCase;
+use Psr\Http\Message\ServerRequestInterface;
+use Sentry\Configuration;
+use Sentry\Event;
+
+abstract class MiddlewareTestCase extends TestCase
+{
+    /**
+     * @param callable                    $middleware
+     * @param Event|null                  $event
+     * @param ServerRequestInterface|null $request
+     * @param \Exception|null             $exception
+     * @param array                       $payload
+     *
+     * @return Event The event returned by the middleware
+     */
+    protected function assertMiddlewareInvokesNext(
```

Comment:
Please change the function definition to make it one-line long to be consistent with the rest of the code where we never enforced a hard limit on code